jenkins/pull_request_creator.py

```diff
@@ -172,11 +174,14 @@ def create(self, delete_old_pull_requests):
     default=True,
     help="If set, delete old branches with the same base branch name and close their PRs"
 )
+@click.option(
+    '--draft', is_flag=True
```

Comment:
I assume `False` is the default default?

Reply:
If it's specified with is_flag, then yes